admission-controller/synchronizer/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "admission-synchronizer",
+  "version": "0.0.2",
+  "description": "",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "start": "node ./dist/index.js",
+    "prod": "npm run build && npm run start"
```

Comment:
Remove these as stated in the above comment.

Reply:
Fixed in https://github.com/kubeshop/monokle-admission-controller/pull/12/commits/b0603be67a691c36612e8d539b6939a267295c63.